CONTRIBUTING.md

```diff
@@ -29,6 +29,7 @@ If you're using webcompat.com and something is confusing, broken, or you think i
 * Search the [issues list](https://github.com/webcompat/webcompat.com/issues) for existing similar issues.  Consider adding to an existing issue if you find one.
 * Choose a descriptive title.
 * Provide a test, snippet of code or screenshot that illustrates the problem. This small gesture goes a long way towards getting a speedy fix.
+* Please always add a `type` label to the issue. If it fits, you also can add a `scope` and / or `language` label. Feel free to take a look at the [label list](https://github.com/webcompat/webcompat.com/labels) first.
```

Comment:
One note: non-collaborators won't be able to add labels to the issue. Maybe we can add some language here to reflect that?

Reply:
That's a good point. 👍